ci/README.md

```diff
@@ -0,0 +1,23 @@
+Keeping CI images up to date
```

Comment:
Perhaps use `#` and `##` for the subheadings?

Reply:
Ok!